### PR TITLE
Fixed Clickhouse issue in CTE 

### LINKED ISSUE #812: Clickhouse: issue in CTE
select query failing on saved snippet.

Screenshot 2023-08-18 at 3 45 20 PM

The below query fails:
```
%%sql
SELECT * from plot_something_subset;
```

Error:

```
DatabaseException: Orig exception: Code: 62.
DB::Exception: Syntax error: failed at position 33 (line 1, col 33): (
SELECT * from plot_something_79c5bc LIMIT 3)
SELECT * from plot_something_subset;. Expected identifier. Stack trace:
```

### PR DESCRIPTION
**Pull request by nooodle-soup**
## Describe your changes
Unable to reproduce errors
Removed pytest xfail for the integration test to check if integration is successful

## Issue number

Closes #812 

## Checklist before requesting a review

- [x] Performed a self-review of my code
- [x] Formatted my code with [`pkgmt format`](https://ploomber-contributing.readthedocs.io/en/latest/contributing/pr.html#linting-formatting)

----
:books: Documentation preview :books:: https://jupysql--885.org.readthedocs.build/en/885/

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "duckdb-engine",
     "pyodbc",
     # sql.plot module tests
-    "matplotlib",
+    "matplotlib==3.7.2",
```
Comment (neelasha23) [inline setup.py:47]: Is there an issue for changing matplotlib version?

Comment (nooodle-soup) [reply setup.py:47]: The latest version of matplotlib breaks the test_magic_plot tests. I discussed this with Eduardo in #878 ([here](https://github.com/ploomber/jupysql/pull/878#issuecomment-1721974298)). I opened an issue regarding that as well (#884).

Comment (neelasha23) [reply setup.py:47]: I think it can be added in the changelog with issue number

Comment (nooodle-soup) [reply setup.py:47]: Should I do it here or in #878 ?

Comment (neelasha23) [reply setup.py:47]: i think we can leave it out . changelog not required

**Review by neelasha23 — APPROVED** [review]

**Review by edublancas — APPROVED** [review]

**Pull request merged**